pkg/internal/checkup/executor/console/login.go

```diff
@@ -29,33 +29,32 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
-func LoginToCentOS(serialConsoleClient vmiSerialConsoleClient, vmiNamespace, vmiName, username, password string) error {
+func (e Expecter) LoginToCentOS(username, password string) error {
```

Comment:
Please consider not using the login function as a method of `Expecter`.

The login function is a consumer of `Expecter`.

Reply:
SafeExpectBatchWithResponse is also a consumer of Expecter, I don't see the difference.
The way I see it - LoginToCentOS is basic function of the console, it does a lot of things in the **same** console process, and breaking it to smaller spawned processes will slow everything down. 

Having said that - we can always experiment with changing it - but I would not want to change such logic in this PR.

Reply:
If `LoginToCentOS()` becomes a method of `Expecter`, do we still need to keep the `login.go` file?

Could you please add reasoning to the commit message about the conversion of this function to a method of `Expecter`?

Reply:
> If LoginToCentOS() becomes a method of Expecter, do we still need to keep the login.go file?

IIUC the original purpose of the `login.go` file was to accommodate different login function for different OSes. In our current checkup we only use one OS, so we could move it back to the original file. 
But having said that - it's not necessary for the purpose of this PR, so in order to not make it more complex - I suggest we do it in another PR.

> Could you please add reasoning to the commit message about the conversion of this function to a method of Expecter?

DONE